package.json

```diff
@@ -2,24 +2,32 @@
   "name": "add_logger",
   "version": "0.0.7",
   "description": "Multichannel File Logger and Console Logger in one",
-  "main": "logger.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+  "main": "./dist/logger.js",
+  "author": "Albert Diones",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/albertdiones/logger/issues"
   },
+  "homepage": "https://github.com/albertdiones/logger#readme",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/albertdiones/logger.git"
   },
   "keywords": [
     "logger"
   ],
-  "author": "Albert Diones",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/albertdiones/logger/issues"
+  "files": [
+    "./dist/*.js"
```

Comment:
bakit js to instead of ts? also non existent directory ./dist/

Reply:
kung i-bbuild mo sya into js then magkakaroon sya ng dist folder and publish it from your project